codegen/templates/vec_mask.rs.tera

```diff
@@ -125,7 +125,49 @@ impl {{ self_t }} {
         {% endif %}
     }
 
-    /// Returns a bitmask with the lowest two bits set from the elements of `self`.
+    /// Creates a vector with all elements set to `v`.
+    #[inline]
+    pub const fn splat(v: bool) -> Self {
+        {% if is_scalar and is_bool %}
```

Comment:
This could call `Self::new` just to reuse existing code, it's `#[inline(always)]` so there shouldn't be any overhead.